google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudStackFrame.java

```diff
@@ -148,7 +148,7 @@ public MyValue(@NotNull Variable variable, @NotNull List<Variable> variableTable
       this.variable = variable.getVarTableIndex() != null ? variableTable
           .get(variable.getVarTableIndex().intValue())
           : variable;
-      members = variable.getMembers();
+      members = this.variable.getMembers();
```

Comment:
Your fix makes sense to me. 

Do you mind explaining the couple lines above this though? What exactly is the `variableTable` and how is its representation of the variable different than the `variable` argument to this constructor? What is the purpose of lines 148-150?